directions/views.py

```diff
@@ -299,6 +299,8 @@ def save(form, filename: str):
                     save(fc_buf, filename=file_contract)
                     pdf_all = BytesIO()
                     inputs = [file_dir, file_contract]
```

Comment:
Вместо такого способа задания, который получается создаёт лишнее присваивание, если only_contract=True, лучше как то так:
```suggestion
                    inputs = [file_contract] if SettingManager.get("only_contract", default='False', default_type='b') else [file_dir, file_contract]
```